src/rust/lib/graph-editor/src/component/node.rs

```diff
@@ -168,6 +175,7 @@ pub struct NodeData {
     pub label  : frp::Dynamic<String>,
     pub events : Events,
     pub view   : component::ShapeView<NodeView>,
+    pub ports  : PortManager,
```

Comment:
lets make it qualified, like `port::Manager`. Btw, I was talking in one of the comments that `Manager` is not the best word as it is like "utils" - everything can be "manager" and everything can be "utils". We try not to use neither of these words in our codebase. What do you think about `port::Registry` instead? :)